src/AstroImages.jl

```diff
@@ -103,12 +102,25 @@ function AstroImage(file::String)
     if ext > 1
        	@info "Image was loaded from HDU $ext"
     elseif ext == 0
-        error("There are no ImageHDU extensions in \"$file\"")
+        error("There are no ImageHDU extensions in '$(fits.filename)'")
     end
-    out = AstroImage(Gray, fits, ext)
-    close(fits)
-    return out
+    return ext
+end
+function AstroImage(files::NTuple{N,String}) where {N}
+    data = [] #Array{Array{Real,2}}(undef,N)     # failing
```

Comment:
Here initializing `Real` is not working. I thought it would work for all `Int` and `Float` types. 
Also, this can contain a mixture of types based on the files given.